docs/logs/README.md

```diff
@@ -107,6 +107,21 @@ Here is the rule of thumb:
   Minutes - Console Application](./getting-started-console/README.md) tutorial
   to learn more.
 
+:heavy_check_mark: You should use dot-separated
```

Comment:
@noahfalk @tarekgh I know this is slightly different from the [current .NET official doc](https://learn.microsoft.com/dotnet/core/extensions/logging#log-category), want to use this PR to facilitate the discussion and agree on the recommendation.

Reply:
This seems fine to me

Reply:
Thanks @noahfalk @tarekgh! Once this is merged, I'll send PR to update the .NET official docs:

* https://learn.microsoft.com/dotnet/core/extensions/logging#log-category
* https://learn.microsoft.com/aspnet/core/fundamentals/logging#log-category

Reply:
.NET official docs updated; it might take some time to have the changes published:

* https://github.com/dotnet/docs/pull/39777
* https://github.com/dotnet/AspNetCore.Docs/pull/31963